src/modules/aws_elb@0.0.1/index.ts

```diff
@@ -56,15 +43,13 @@ export const AwsElbModule: Module = new Module({
       out.loadBalancer = ctx.memo?.db?.AwsListener?.[l.LoadBalancerArn] ?? await AwsElbModule.mappers.loadBalancer.db.read(ctx, l?.LoadBalancerArn);
       out.port = l?.Port;
       out.protocol = l?.Protocol as ProtocolEnum;
-      out.defaultActions = await Promise.all(l.DefaultActions?.map(async a => {
-        const dbAct = await ctx.orm.findOne(AwsAction, {
-          where: {
-            targetGroup: { targetGroupArn: a.TargetGroupArn },
-          },
-          relations: ['targetGroup'],
-        });
-        if (!dbAct) return AwsElbModule.utils.actionMapper(a, ctx);
-        return dbAct;
+      await Promise.all(l.DefaultActions?.map(async a => {
```

Comment:
Should this be refactored into a `for await` loop instead?

Reply:
It could yes, probably better